screen/loans/src/main/java/com/ivy/loans/loan/LoanViewModel.kt

```diff
@@ -300,18 +301,28 @@ class LoanViewModel @Inject constructor(
         } ?: defaultCurrencyCode
     }
 
-    private suspend fun calculateAmountPaid(loan: Loan): Double {
+    /**
+     *  Calculates the total amount paid and the total loan amount including any changes made to the loan.
+     *  @return A Pair containing the total amount paid and the total loan amount.
+     */
+    private suspend fun calculateAmountPaidAndTotalAmount(loan: Loan): Pair<Double, Double> {
```

Comment:
I don't see why not. 

When this gets called on the loan created before the merge of this PR. It will return `loan.amount` which is the value the loan was created with. It will simply never go into `INCREASE` block.

Reply:
Makes sense. Thanks for the clarification!